### PR TITLE
Update out of date README.md

### PR DESCRIPTION
**Pull request by QuanMPhm**
Updated the README, after the #14 Lenovo PR is merged

### CODE REVIEW

**Comment by knikolla** [comment]
In general, since this only updated the README file, you didn't have to make it on top of the Lenovo PR since it had no actual dependency on it, besides it being merged before. Now that Lenovo has merged please rebase.

**Comment by QuanMPhm** [comment]
@knikolla I have rebased the README commit

File: README.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+- Project - Allocation
+- Manager (PI)
+- Institution
+- SU Hours (GBhr or SUhr)
+- SU Type
+- Cost
+
+```
+usage: process_report.py [-h] --pi-file PI_FILE --projects-file PROJECTS_FILE --timed-projects-file TIMED_PROJECTS_FILE [--output-file OUTPUT_FILE]
+                         [--output-folder OUTPUT_FOLDER] [--HU-invoice-file HU_INVOICE_FILE] [--HU-BU-invoice-file HU_BU_INVOICE_FILE] [--old-pi-file OLD_PI_FILE]
```
Comment (naved001) [inline README.md:21]: can you add how a sample OLD_PI_FILE would look like? 

The non-billable section of this document shows sample files for pi.txt, projects.txt etc.

Comment (QuanMPhm) [reply README.md:21]: Done

**Review by naved001 — APPROVED** [review]

**Pull request merged**